### DISCUSSION TITLE
MacintoshPi

### DISCUSSION THREAD

**Discussion by symbios24** [Exarkuniv/RetroPie-Extra#217] [Ideas]
Hi there

Is it possible to add this:

https://github.com/jaromaz/MacintoshPi

git clone https://github.com/jaromaz/MacintoshPi
cd MacintoshPi
./build_all.sh

dependencies and required packages will be installed automatically:

Thank you

**Comment by Exarkuniv** [comment]
makeing a script is a no. it needs to be ran as pi user. and there is no easy why to make this happen in a script
